frontend/test/xray/xray.integ.spec.js

```diff
@@ -332,7 +347,7 @@ describe("xray integration tests", () => {
 
             app = mount(store.getAppContainer())
 
-            await store.waitForActions([LOAD_CURRENT_USER, INITIALIZE_SETTINGS])
+            await store.waitForActions([LOAD_CURRENT_USER, INITIALIZE_SETTINGS], { timeout: 20000 })
```

Comment:
The timeout seems pretty long here. Might be worth commenting on?

Reply:
I think that the timeout is just unnecessary cruft caused by a copy-paste in this case